### PR TITLE
add event name for ping webhooks in the header

### PR DESCRIPTION
**Pull request by fantasticfears**

### CODE REVIEW

**Comment by tgxworld** [comment]
Tests please :grin: Just a simple one to cover this case will do.

**Comment by fantasticfears** [comment]
Done!

File: spec/controllers/admin/web_hooks_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  context 'while logged in as an admin' do
+    before do
+      @user = log_in(:admin)
+    end
+    let(:web_hook) { Fabricate(:web_hook) }
+
+    describe '.create' do
+      it 'creates a webhook' do
+        xhr :post, :create, web_hook: {
+            payload_url: 'https://meta.discourse.org/',
```
Comment (tgxworld) [inline spec/controllers/admin/web_hooks_controller_spec.rb]: Style nitpick: The indentation here is different than the one used below.

File: spec/controllers/admin/web_hooks_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    expect(Admin::WebHooksController < Admin::AdminController).to eq(true)
+  end
+
+  context 'while logged in as an admin' do
+    before do
+      @user = log_in(:admin)
+    end
+    let(:web_hook) { Fabricate(:web_hook) }
+
+    describe '.create' do
```
Comment (tgxworld) [inline spec/controllers/admin/web_hooks_controller_spec.rb]: Controller methods are instance methods so this should be '#create'

File: spec/controllers/admin/web_hooks_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+          category_ids: []
+        }
+        expect(response.status).to eq 422
+        response_body = JSON.parse(response.body)
+
+        expect(response_body["errors"]).to be_present
+      end
+    end
+
+    describe '.ping' do
```
Comment (tgxworld) [inline spec/controllers/admin/web_hooks_controller_spec.rb]: Ditto https://github.com/discourse/discourse/pull/4841/files#r114234349

**Comment by tgxworld** [comment]
Thanks @fantasticfears

**Pull request merged**